clients/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -276,7 +276,8 @@ describe("Dataset", () => {
       cy.getByTestId("connect-db-btn");
     });
 
-    it("Can create a dataset via yaml", () => {
+    // TODO: Update to include the @monaco-editor/react component
```

Comment:
> How much effort should we put into the Cypress testing specifically updating the datasets.cy.ts file. Changes are needed to reflect the @monaco-editor/react NPM package. Initially, I ran into issues emulating copy/paste functionality with integrating the @monaco-editor/react NPM package. I updated the datasets.cy.ts file with TODO statements temporarily to function as a placeholder. Thoughts?

It looks like `window.monaco` still gets defined when using the react component, which means we can interact with monaco from `cy.window(...)`. Using [window.monaco.editor.setValue](https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneCodeEditor.html#setValue) might work, for example.

I think interacting with a package like this is a place we can get a lot of value from automated tests: it's slow to manually go in verify that typing a big block of text still works, so it might get left out of manual review.

Reply:
We can create a follow up ticket to get this working and push this through for now.